src/components/TopBar/components/Nav.tsx

```diff
@@ -1,28 +1,70 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { keyframes } from "styled-components";
 import StyledRouterLink from "./StyledRouterLink";
 import StyledLink from "./StyledLink";
 import StyledNestedLink from "./StyledNestedLink";
 
-const Nav: React.FC = () => {
+interface NavProps {
+  onDismiss: () => void;
+  mobileMenu?: boolean;
+}
+
+const Nav: React.FC<NavProps> = ({ onDismiss, mobileMenu }) => {
   return (
-    <StyledNav>
-      <StyledRouterLink target="/dashboard" label="Dashboard" />
-      <StyledNestedLink target="/umbrella" label="Projects" type="router" >
-        <StyledRouterLink target="/umbrella" label="Umbrella" style={{ height: 26, display: "flex", alignItems: "center" }} />
-        <StyledLink href="https://degenerative.finance/" label="Degenerative" style={{ height: 26, display: "flex", alignItems: "center" }} />
-      </StyledNestedLink>
-      <StyledRouterLink target="/governance" label="Govern" />
-      <StyledRouterLink target="/farm" label="Farm" />
-      <StyledRouterLink target="/migrate" label="Migrate" />
-      <StyledLink href="https://docs.yam.finance/" label="FAQ" />
-    </StyledNav>
+    <>
+      { mobileMenu && (
+        <StyledBackdrop onClick={onDismiss} />
+      )}
+      <StyledNav>
+        <StyledRouterLink target="/dashboard" label="Dashboard" onDismiss={onDismiss} />
+        <StyledNestedLink target="/umbrella" label="Projects" type="router" mobileMenu={mobileMenu} >
+          <StyledRouterLink target="/umbrella" label="Umbrella" style={ mobileMenu ? {} : { height: 26, display: "flex", alignItems: "center" }} onDismiss={onDismiss} />
+          <StyledLink href="https://degenerative.finance/" label="Degenerative" style={mobileMenu ? {} : { height: 26, display: "flex", alignItems: "center" }} onDismiss={onDismiss} />
+        </StyledNestedLink>
```

Comment:
something doesnt look right on the nested menus on small devices: ![131231](https://user-images.githubusercontent.com/2432272/108431408-424f8f00-725c-11eb-9de0-b9a5031b1d00.PNG)